internal/bminventory/inventory.go

```diff
@@ -999,11 +999,14 @@ func (b *bareMetalInventory) InstallCluster(ctx context.Context, params installe
 			log:    log,
 			params: params,
 		}
-		err = b.db.Transaction(cInstaller.install)
-		if err == nil {
-			//send metric when the installation process has been started
-			b.metricApi.InstallationStarted(cluster.OpenshiftVersion, *cluster.ID)
+		if err = b.db.Transaction(cInstaller.install); err != nil {
+			return
 		}
+
+		// send metric and event that installation process has been started
+		b.metricApi.InstallationStarted(cluster.OpenshiftVersion, *cluster.ID)
+		msg := fmt.Sprintf("Updated status of cluster %s to installing", cluster.ID)
+		b.eventsHandler.AddEvent(ctx, *cluster.ID, nil, models.EventSeverityInfo, msg, time.Now())
```

Comment:
No need for msg variable, just add the message to the AddEvent call

Reply:
done